### PR TITLE
Add Deta topic

### PR DESCRIPTION
**Pull request by berrysauce**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---------------------------------------------------------------------

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> A lot of projects use this service and curating this topic would inform users better about it.

---------------------------------------------------------------------

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Review by wes9971 — APPROVED** [review]
Ok cool

**Comment by Bonnie1z** [comment]
I will follow all instructions. I'm new to this and I'm in the process of
learning. So can you simplify what I should do?

**Comment by Stefan-hoppe** [comment]
> ### Thank you for contributing! Please confirm this pull request meets the following requirements:
> 
> - [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
> - [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).
> 
> ### Which change are you proposing?
> 
>   - [ ] Suggesting edits to an existing topic or collection
>   - [x] Curating a new topic or collection
>   - [ ] Something that does not neatly fit into the binary options above
> 
> ---------------------------------------------------------------------
> 
> ### Curating a new topic or collection
> 
> - [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
> - [x] My folder contains a `*.png` image (if applicable) and `index.md`
> - [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs
> 
> > A lot of projects use this service and curating this topic would inform users better about it.
> 
> ---------------------------------------------------------------------
> 
> **Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**
> 

Hirnlos wie die anderen hefties alle lachen und das Lebt weiter an ihrer Seite Gedanken loss

File: topics/deta/index.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+created_by: Abstract Computing UG in Berlin
+display_name: Deta
+github_url: https://github.com/deta/
+logo: deta.png
+short_description: A Cloud for the Next Billion Ideas. Build your apps in hours, deploy them in seconds.
+topic: deta
+url: https://www.deta.sh/
+---
+Deta makes serverless development easy. They offer free serverless hosting and NoSQL databases. Check out their Python, JavaScript, and Go SDKs for optimal usage.
```
Comment (MikeMcQuaid) [inline topics/deta/index.md:10]: Could this be reworded to sound a bit less like advertising? Thanks ❤️

Comment (berrysauce) [reply topics/deta/index.md:10]: Sure, I'll try!

Comment (berrysauce) [reply topics/deta/index.md]: Done!

**Comment by MikeMcQuaid** [comment]
Wording looks good but image needs resized: https://github.com/github/explore/pull/2564/checks?check_run_id=3622017109#step:7:380

**Comment by smashwilson** [comment]
Thanks!

**Pull request merged**